docs/setting-up.md

```diff
@@ -50,15 +50,31 @@ More information can be found at [this documentation](https://help.github.com/ar
 
    # Linux/OS X
    ./install.sh --path-update true
+
    # Windows
    install.bat --path-update true
+
+   # Red Hat/CentOS/Fedora
+   sudo yum install google-cloud-sdk
+
+   # Debian/Ubuntu
+   sudo apt-get install google-cloud-sdk
```

Comment:
No need to include them for the following reasons:
- They are mentioned in the official docs
- If you only display them like this, it's unlikely to work because the "Before you begin" steps are not done

If anything, just do what I suggested: "If you use Debian/Ubuntu/Red Hat/CentOS, the installation is slightly more complicated. Refer to the official documentation provided by Google."

Reply:
Okay :+1: